src/main/java/com/conveyal/taui/controllers/SinglePointAnalysisController.java

```diff
@@ -54,7 +56,11 @@
             brokerRes = HttpUtil.httpClient.execute(post);
             res.status(brokerRes.getStatusLine().getStatusCode());
             res.type(brokerRes.getFirstHeader("Content-Type").getValue());
-            // TODO set encoding?
+            //FIXME this is a hack for geotiff exports that should be fixed when broker changes are implemented
+            if (req.headers("Accept").equals("image/tiff")) {
+                res.header("Content-Type","application/octet-stream");
```

Comment:
Can we add some more detail about why we want to force it to `application/octet-stream` instead of `image/tiff`, and why we are gzipping a tiff file instead of (or in addition to) setting LZW compression on the TIFF file itself? I don't really understand these points myself. I guess the first one is just because the UI doesn't have a handler for `image/tiff`?

Reply:
Generally I think we should avoid "this is a hack" kind of comments and always spell out what the hack is and what it's meant to be replaced with.